packages/utils/src/misc.ts

```diff
@@ -236,6 +236,18 @@ export function formatIdTarget(id: number): string {
   return formatExpirerTarget("id", id);
 }
 
+export function parseExpirerTarget(target: string) {
+  const [type, value] = target.split(":");
+  const parsed: { id?: number; topic?: string } = { id: undefined, topic: undefined };
+  if (type === "topic" && typeof value === "string") {
+    parsed.topic = value;
+  } else if (type === "id" && typeof value === "number") {
+    parsed.id = value;
+  }
+
```

Comment:
Do we need to handle the unknown case here too? 

E.g. throw if `type` is a garbage value or `id` is a string and doesn't match the second statement, so the function calling `parseExpirerTarget` doesn't just proceed with an empty object.

Reply:
Good idea, in fact it made me catch another bug with `typeof value === "number"` it will never be true because id here will always be string from `const [type, value] = target.split(":");` so needs to be converted to Number